server/dive_tasks/tasks.py

```diff
@@ -677,7 +677,12 @@ def convert_large_images(self: Task, folderId, user_id: str, user_login: str):
     ]
     for item in items_to_convert:
         # Assumes 1 file per item
-        gc.post(f'item/{item["_id"]}/tiles')
+        try:
+            gc.get(f'item/{str(item["_id"])}/tiles')
+            manager.write(f'Skipping {item["name"]}, already a large image\n')
+            continue
+        except Exception:
```

Comment:
What if `gc.get` throws an exception unrelated to the large image item existing yet or not? Is there a specific kind of exception that would be idea to capture here?